projects-to-be-tested/maven/spring/layered-jar-disabled/pom.xml

```diff
@@ -0,0 +1,84 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright (c) 2019 Red Hat, Inc.
+    This program and the accompanying materials are made
+    available under the terms of the Eclipse Public License 2.0
+    which is available at:
+
+        https://www.eclipse.org/legal/epl-2.0/
+
+    SPDX-License-Identifier: EPL-2.0
+
+    Contributors:
+      Red Hat, Inc. - initial API and implementation
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.eclipse.jkube.integration-tests</groupId>
+    <artifactId>jkube-integration-tests-project</artifactId>
+    <version>${revision}</version>
+    <relativePath>../../../../pom.xml</relativePath>
+  </parent>
+
+  <artifactId>spring-boot-layered-jar-disabled</artifactId>
+  <name>${global.name} :: Spring Boot :: Layered Jar Disabled</name>
+  <description>
+    Spring Boot with Explicit Configuration to disable Layered Jar
+  </description>
+
+  <properties>
+    <jkube.build.strategy>docker</jkube.build.strategy>
+    <jkube.enricher.jkube-service.type>NodePort</jkube.enricher.jkube-service.type>
+  </properties>
```

Comment:
How is the layered Jar being disabled?
Related to why do we have a kubernetes-maven-plugin plugin dependency?

Reply:
Layered Jar is disabled in Spring Boot Maven Plugin Configuration.

You're right about Kubernetes Maven Plugin, it's not used. I'll remove it.